docs/tutorial/get-started.md

```diff
@@ -634,6 +657,43 @@ This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perf
 
           </details>
 
+   1. Remove
+
+      - Remove indexed 400 training datasets from the Vald agent.
+          <details><summary>example code</summary><br>
+
+        ```go
+        for i := range ids [:insertCount] {
+            _, err := client.Remove(ctx, &payload.Remove_Request{
+                Id: &payload.Object_ID{
+                    Id: ids[i],
+                },
+            })
+            if err != nil {
+                glg.Fatal(err)
+            }
+            if i%10 == 0 {
+                glg.Infof("Removed %d", i)
+            }
+        }
+        ```
+
+          </details>
+
+
+      - Remove manually instead of waiting for auto indexing.
+        If you run below code, the indexes will be removed from the Vald Agent and the Backup file.
```

Comment:
I through the eyes of the user, I thought that `CreateIndex` would mean deleting the file, so what about the following?.
```suggestion
        If you run below code, the indexes will be removed from the Vald Agent and the Backup file because you deleted all the inserted vectors in the above section.
```

Reply:
btw, Should we change from `SaveIndex ` to `SaveAndCreateIndex` method.
Will the backup file be erased, but will the indexing data be erased as well?

Reply:
@hlts2 @kevindiu 
I think `CreateAndSaveIndex()` won't work well.
cuz, there is no index to create and save.
I just tried to use `CreateAndSaveIndex()` instead of the current implementation, but end with the error `FailedPreCondition desc = CreateAndSaveIndex API failed to create indexes pool_size = 400`.
If you can, could you please try it?

Reply:
@hlts2 @kevindiu 
I should announce to guys that the current `vald-agent=ngt:latest` image will not work well, so if you'd like to confirm the tutorial, please use the `nightly` version.
The latest version will be updated when released next version.


And also, `CreateAndSaveIndex()` is good. but, this time, I'd like to another method as `SaveIndex()` instead of this to show vald client does not have only `CreateAndSaveIndex()`.
How do you think about it?